### PR TITLE
Views and styling

### PR DESCRIPTION
**Pull request by LoviMadaan**
### **Milestone: Views**
**The following changes are being introduced:**

- [x] Added views associated with models and controllers
- [x] Implemented the design from the sneak peek wireframes
- [x] Minimal styling is done. Emphasis is on functionality
- [x] No login logic implemented.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -4,6 +4,6 @@ def index
   end
 
   def show
-    @user = User.find_by_id(params[:id])
+    @user = User.find(params[:id])
```
Comment (brainconnect93) [inline app/controllers/users_controller.rb:7]: - [ ] Hi @LoviMadaan, Great effort working on your project but currently when I try to add some data to the database using the `rails c` command i.e add `user` and `post` it wasn't adding but I could add only a comment. I would like you to check the video below for my claim.

Video claim 👇 


https://github.com/LoviMadaan/Simple_blog/assets/78634841/d4993bec-ceac-4d76-8b7e-8d3402efa8e1

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @LoviMadaan, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
- Views and controller properly implemented ✔️
- Good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @LoviMadaan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Nice job addressing the comments made from the previous review :rocket: 
Screenshot from 2023-07-08 13-34-45
Screenshot from 2023-07-08 13-33-29



 ## Required Changes ♻️
_Check the comments under the review._
 - [ ] It would be better if clicking on the `See all post` link opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png). However, we are currently getting error displaying the right page as we can see in the screenshot below
Screenshot from 2023-07-08 13-40-06


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @LoviMadaan, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜  
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**